src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -216,23 +231,34 @@ protected void end() {
 
     public Command align() {
         return new Command("align", this) {
-
-            NetworkTableInstance inst = NetworkTableInstance.getDefault();
-            NetworkTable table = inst.getTable("Vision Correction Table");
             double visionCorrectionFactor = table.getEntry("Vision Correction").getDouble(0);
+            boolean visionConfirmation = table.getEntry("Vision Found").getBoolean(false);
```

Comment:
Suggest storing the entry, and then doing `getBoolean` in execute

Reply:
done with ben